src/Logging/Logging.Batching/BatchLogService.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Concurrent;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)